lib/engine/alerts.ex

```diff
@@ -30,6 +46,7 @@ defmodule Engine.Alerts do
         stop_ids
       ) do
     stop_ids
+    |> Enum.reject(&MapSet.member?(@platform_stop_ids, &1))
     |> Enum.map(&stop_status(tables.stops_table, &1))
```

Comment:
There is a potential error here if every `stop_id` in the `Enum` is included in `@platform_stop_ids`. However, this will never happen in actuality given the list of current `@platform_stop_ids`, and should not arise in the future regardless based on how/why these special case `stop_ids` are configured